pydash/numerical.py

```diff
@@ -117,6 +119,29 @@ def average(collection, callback=None):
 mean = average
 
 
+def call_math_operator(value1, value2, op, default):
```

Comment:
Generally, the helper functions are defined alllll the way at the bottom: https://github.com/dgilland/pydash/blob/8ca3261ba7a8a28c681268df48d23b399774312b/pydash/numerical.py#L515

So if you're moving the function now, best to put it there.